### PR TITLE
[TW-1100] Flows minimap grouping qol

### PR DESCRIPTION
**Pull request by petewitcher-postman**
Added content about grouping, renaming, and resizing blocks. Also updated several screenshots to include the minimap and other UI changes.

### CODE REVIEW

File: src/pages/docs/postman-flows/gs/blocks-information.md [hunk 1]
```diff
@@ -83,3 +84,23 @@ There are three types of blocks:
 ## Passing information
 
 In Flows, information passes from one block to the next. This information could be a text, a number, or it could be larger complex information like a list of users, or an entire movie.
+
+## Working with blocks
+
+You can resize, rename, and group blocks together in Postman Flows.
+
+To resize a block, select and drag the resizing handle in the block's lower right corner.
```
Comment (MatthewPinsker-postman) [inline src/pages/docs/postman-flows/gs/blocks-information.md:92]:
```suggestion
To resize a block, select and drag the resizing handle in the block's lower-right corner.
```

File: src/pages/docs/postman-flows/gs/blocks-information.md [hunk 2]
```diff
@@ -83,3 +84,23 @@ There are three types of blocks:
...
+
+<img alt="Resize a block" src="https://assets.postman.com/postman-docs/v10/flows-resize-block-v10.jpg" width="300px">
+
+To rename a block, hover over the title and select the edit icon, then enter the block's new name.
+
+<img alt="Rename a block" src="https://assets.postman.com/postman-docs/v10/flows-rename-block-v10.jpg" width="300px">
+
+To group blocks together, press and hold **⌘** or **Ctrl**, select the names of the blocks you want to group, then select the **Group selection** icon.
```
Comment (MatthewPinsker-postman) [inline src/pages/docs/postman-flows/gs/blocks-information.md:100]:
```suggestion
To group blocks together, press and hold **⌘** or **Ctrl**, select the names of the blocks you want to group, then select the group selection icon.
```

File: src/pages/docs/postman-flows/gs/blocks-information.md [hunk 3]
```diff
@@ -83,3 +84,23 @@ There are three types of blocks:
...
+
+![Group blocks](https://assets.postman.com/postman-docs/v10/flows-group-blocks-v10-1.gif)
+
+To ungroup blocks, hover over the group and select the **Ungroup** icon.
```
Comment (MatthewPinsker-postman) [inline src/pages/docs/postman-flows/gs/blocks-information.md:104]: Same feedback as above. I'm wondering if we could use this preexisting icon image: `Close icon`

File: src/pages/docs/postman-flows/tutorials/building-your-first-flow.md [hunk 4]
```diff
@@ -50,27 +50,27 @@ This quick start tutorial creates a flow to check if a website is down or not. I
...
 
 1. In the **Send Request** block, select **Add request &gt; DownDetector &gt; sitemap**. Then select the port next to **Success**, and drag and drop it to the right. Select **Evaluate** from the list that appears to add an **Evaluate** block.
 
-    ![Add an **Evaluate** block](https://assets.postman.com/postman-docs/v10/flow-tut-add-eval-v10.gif)
+    ![Add an **Evaluate** block](https://assets.postman.com/postman-docs/v10/flow-tut-add-eval-v10-1.gif)
 
 1. In the **Evaluate** block, select `value1` and change it to `status`. Then select **Enter path...**, scroll down, and select **http.status**.
 
-    ![Rename value1 and select `http.status`](https://assets.postman.com/postman-docs/v10/flow-tut-select-status-v10.gif)
+    ![Rename value1 and select `http.status`](https://assets.postman.com/postman-docs/v10/flow-tut-select-status-v10-1.gif)
```
Comment (MatthewPinsker-postman) [inline src/pages/docs/postman-flows/tutorials/building-your-first-flow.md:65]: Should we remove the code sample styling from the Alt text? I'm not sure if the code styling shows up if the image link breaks. If it does, my thinking is that users might confuse it for instructions. Also, I'm not sure how screen readers handle the styling.

File: src/pages/docs/postman-flows/tutorials/building-your-first-flow.md [hunk 5]
```diff
@@ -50,27 +50,27 @@ This quick start tutorial creates a flow to check if a website is down or not. I
...
 
 1. Right-click the canvas and select **Send Request** from the list of blocks. Connect the **Send Request** block to the **Start** block.
 
-    ![Create a **Send Request** block and connect it](https://assets.postman.com/postman-docs/v10/flow-tut-add-send-v10.gif)
+    ![Create a **Send Request** block and connect it](https://assets.postman.com/postman-docs/v10/flow-tut-add-send-v10-1.gif)
 
 1. In the **Send Request** block, select **Add request &gt; DownDetector &gt; sitemap**. Then select the port next to **Success**, and drag and drop it to the right. Select **Evaluate** from the list that appears to add an **Evaluate** block.
 
-    ![Add an **Evaluate** block](https://assets.postman.com/postman-docs/v10/flow-tut-add-eval-v10.gif)
+    ![Add an **Evaluate** block](https://assets.postman.com/postman-docs/v10/flow-tut-add-eval-v10-1.gif)
```
Comment (MatthewPinsker-postman) [inline src/pages/docs/postman-flows/tutorials/building-your-first-flow.md:61]: Same feedback as below comments.

File: src/pages/docs/postman-flows/tutorials/building-your-first-flow.md [hunk 6]
```diff
@@ -50,27 +50,27 @@ This quick start tutorial creates a flow to check if a website is down or not. I
...
 1. Select **Start writing an FQL query...** and enter `status=200`.
 
-    ![Enter `status=200`](https://assets.postman.com/postman-docs/v10/flow-tut-status-200-v10.gif)
+    ![Enter `status=200`](https://assets.postman.com/postman-docs/v10/flow-tut-status-200-v10-1.gif)
 
-1. Drag and drop the port on the right side of the **Evaluate** block and add an **Output** block.
+1. Drag and drop the port on the right side of the **Evaluate** block and add an **Output** block. Select **Run**. The **Output** block displays **True**, which means the website is up and running.
 
-1. Select **Run**. The **Output** block displays **True**, which means the website is up and running.
+    ![Add an **Output** block and select **Run**](https://assets.postman.com/postman-docs/v10/flow-tut-output-run-v10.gif)
```
Comment (MatthewPinsker-postman) [inline src/pages/docs/postman-flows/tutorials/building-your-first-flow.md:73]: Same feedback as above comments.

File: src/pages/docs/postman-flows/tutorials/building-your-first-flow.md [hunk 7]
```diff
@@ -107,53 +107,53 @@ This is the beginning of a detailed tutorial that builds a flow that gets a list
...
 
 1. Create another **Select** block and connect it to the **Success** port on the **Send Request** block.
 
-    ![Add another Select block](https://assets.postman.com/postman-docs/v10/flow-add-another-select-1-v10.jpg)
+    ![Add another Select block](https://assets.postman.com/postman-docs/v10/flow-add-another-select-2-v10.jpg)
 
 1. In the **Select** block you created, select **Enter path...** and select `body.next` to get the link to the next page of results.
 
-    ![Select `next`](https://assets.postman.com/postman-docs/v10/flow-select-next-v10.jpg)
+    ![Select `next`](https://assets.postman.com/postman-docs/v10/flow-select-next-1-v10.jpg)
```
Comment (MatthewPinsker-postman) [inline src/pages/docs/postman-flows/tutorials/building-your-first-flow.md:146]: Same feedback as below comments.

File: src/pages/docs/postman-flows/tutorials/building-your-first-flow.md [hunk 8]
```diff
@@ -107,53 +107,53 @@ This is the beginning of a detailed tutorial that builds a flow that gets a list
...
 
 1. Create an **If** block to the right of the `body.next` **Select** block. Connect the `body.next` **Select** block to the `key` port. This creates a variable named `value1`.
 
 1. Also connect the `body.next` **Select** block to the `Data` port in the **If** block.
 
-    ![Connect the **If** block](https://assets.postman.com/postman-docs/v10/flow-add-if-block-2-v10.jpg)
+    ![Connect the **If** block](https://assets.postman.com/postman-docs/v10/flow-add-if-block-3-v10.jpg)
```
Comment (MatthewPinsker-postman) [inline src/pages/docs/postman-flows/tutorials/building-your-first-flow.md:152]: Same feedback as above comments.

File: src/pages/docs/postman-flows/tutorials/building-your-first-flow.md [hunk 9]
```diff
@@ -107,53 +107,53 @@ This is the beginning of a detailed tutorial that builds a flow that gets a list
...
 
 1. In the **If** block, select `value1` and replace it with `has_next`. This assigns the `body.next` value from the **Select** block to the `has_next` variable in the **If** block.
 
-    ![Enter `has_next`](https://assets.postman.com/postman-docs/v10/flow-enter-has-next-1-v10.jpg)
+    ![Enter `has_next`](https://assets.postman.com/postman-docs/v10/flow-enter-has-next-2-v10.jpg)
```
Comment (MatthewPinsker-postman) [inline src/pages/docs/postman-flows/tutorials/building-your-first-flow.md:156]: Same feedback as above comments.

File: src/pages/docs/postman-flows/tutorials/building-your-first-flow.md [hunk 10]
```diff
@@ -172,5 +172,5 @@ After completing the above steps, select **Run**. The flow runs and logs all the
 
 <!-- vale Postman.Vocab = YES -->
 
-![Watch it run](https://assets.postman.com/postman-docs/v10/flow-watch-it-run-2-v10.gif)
+![Watch it run](https://assets.postman.com/postman-docs/v10/flow-watch-it-run-3-v10.gif)
```
Comment (MatthewPinsker-postman) [inline src/pages/docs/postman-flows/tutorials/building-your-first-flow.md:175]: Should the GIF have a border?

File: src/pages/docs/postman-flows/tutorials/building-your-first-flow.md [hunk 11]
```diff
@@ -50,27 +50,27 @@ This quick start tutorial creates a flow to check if a website is down or not. I
 
 1. Go to the [**DevOps Flows**](https://www.postman.com/postman/workspace/devops-flows) workspace and [fork](/docs/collaborating-in-postman/using-version-control/forking-entities/) the [**DownDetector**](https://www.postman.com/postman/workspace/devops-flows/collection/23919558-2ea958e4-8647-4b79-859a-47a2c7608b4d?action=share&creator=23919558) collection to your workspace. Then, in your workspace, select **New &gt; Flows** and [hide the sidebar](/docs/getting-started/navigating-postman/#sidebar).
 
-    ![Fork the collection](https://assets.postman.com/postman-docs/v10/flow-tut-fork-collection-1-v10.gif)
+    ![Fork the collection](https://assets.postman.com/postman-docs/v10/flow-tut-fork-collection-3-v10.gif)
```
[the remark this answers is not in the document]
Comment (petewitcher-postman) [reply src/pages/docs/postman-flows/tutorials/building-your-first-flow.md:53]: No gifs don't have borders

File: src/pages/docs/postman-flows/tutorials/building-your-first-flow.md (hunk 10 above)
Comment (petewitcher-postman) [reply src/pages/docs/postman-flows/tutorials/building-your-first-flow.md:175]: Negative

File: src/pages/docs/postman-flows/tutorials/building-your-first-flow.md (hunk 11 above)
[the remark this answers is not in the document]
Comment (MatthewPinsker-postman) [reply src/pages/docs/postman-flows/tutorials/building-your-first-flow.md:53]: There's a bullet point about adding borders to GIFs in the [Screenshot formatting section](https://postmanlabs.atlassian.net/wiki/spaces/TW/pages/3155165686/Image+guidelines+for+Learning+Center+docs#Screenshot-formatting).

[the remark this answers is not in the document]
Comment (petewitcher-postman) [reply src/pages/docs/postman-flows/tutorials/building-your-first-flow.md:53]: Ah thanks I had not seen that

File: src/pages/docs/postman-flows/tutorials/webhooks.md [hunk 12]
```diff
@@ -91,15 +91,15 @@ In the environment you just forked, paste your polygon API key where it says **Y
...
+![Create Send Request block](https://assets.postman.com/postman-docs/v10/make-send-request-block-v10.gif)
 
 The final step is to send all this information back to the Slack app. First, since slack requires the text sent to them to be in JSON format, create a **Record** block, name the key **text** and add a **Select** block. You can either use the dropdown menu to select, or enter ```body.results.0.c``` which is how polygon stores their stock quotes. Now create a new **Send Request** block, drag the output of the **Record** block to the **message_to_slack** variable which is what we're going to send to the app, and drag a connection from the second **Evaluate** block (that holds the destination in Slack to reply to) to the **slack_hook_url** input of the **Send Request** block.
 
-![Create Send Request block](https://assets.postman.com/postman-labs-docs/cloud-execution/sending-to-slack.gif)
+![Create Send Request block](https://assets.postman.com/postman-docs/v10/sending-to-slack-v10-1.gif)
 
 All that's left is hitting the **Deploy** button and trying it out in Slack!
```
Comment (MatthewPinsker-postman) [inline src/pages/docs/postman-flows/tutorials/webhooks.md:104]:
```suggestion
All that's left is selecting **Deploy** and trying it out in Slack!
```

**Review by MatthewPinsker-postman — APPROVED** [review]
LGTM! I added a few suggestions. Thanks!

File: src/pages/docs/postman-flows/gs/the-interface.md [hunk 13]
```diff
@@ -64,38 +65,42 @@ The toolbar gives you access to tools you can use to manipulate the canvas.
...
 
 The right sidebar is the place where you'll see more information about your flow and its current state.
-![Right sidebar](https://assets.postman.com/postman-labs-docs/interface/updated-interface-context-bar.png)
+![Right sidebar](https://assets.postman.com/postman-docs/v10/updated-interface-context-bar-v10-3.jpg)
 
-* **Element info**
+* **Runs**
 
-  You can see the specification for the selected block here. It has the input, output, and variable descriptions.
+  Shows deployment status and your flow's webhook URL. Enables you to deploy or undeploy your flow. You can toggle between viewing the deployed flow and viewing the flow in the editor. Also shows webhook events for the selected flow.
```
Comment (akinard-postman) [inline src/pages/docs/postman-flows/gs/the-interface.md:83]:
```suggestion
  Shows the deployment status and your flow's webhook URL. Enables you to deploy or undeploy your flow. You can toggle between viewing the deployed flow and viewing the flow in the editor. Also shows the selected flow's webhook events.
```

File: src/pages/docs/postman-flows/gs/the-interface.md [hunk 14]
```diff
@@ -64,38 +65,42 @@ The toolbar gives you access to tools you can use to manipulate the canvas.
...
-  When your flow encounters an error while running, you can read the logs in the **Execution issues** pane.
+  Shows the available configuration options and ports for the selected block. You can edit the block's title here.
 
-  ![execution errors](https://assets.postman.com/postman-labs-docs/interface/updated-interface-execution-error-eval-block.gif)
+  ![Configuration](https://assets.postman.com/postman-docs/v10/flows-right-sidebar-config-v10.jpg)
 
 * [**Forks**](/docs/collaborating-in-postman/using-version-control/forking-entities/)
 
-  A list of all the forks of the flow that have been created.
+  Shows a list of all the forks of the flow that have been created.
```
Comment (akinard-postman) [inline src/pages/docs/postman-flows/gs/the-interface.md:95]:
```suggestion
  Shows a list of all the created forks of the flow.
```

File: src/pages/docs/postman-flows/tutorials/passing-data-between-blocks.md [hunk 15]
```diff
@@ -238,19 +238,19 @@ You can extract specific values from response data in a number of ways using var
 
 1. Select the **Language** dropdown list and select **JSON**.
 
-    ![Select JSON](https://assets.postman.com/postman-docs/v10/flow-passing-data-start-1-v10.jpg)
+    ![Select JSON](https://assets.postman.com/postman-docs/v10/flow-passing-data-start-5-v10.jpg)
 
 1. Connect an **Evaluate** block to the **Start** block.
-1. In the **Evaluate** block, select **key** and enter `body`. This assigns all the data received from the **Start** block to the variable `body`.
+1. In the **Evaluate** block, select **value1** and enter `body`. This assigns all the data received from the **Start** block to the variable `body`.
```
Comment (akinard-postman) [inline src/pages/docs/postman-flows/tutorials/passing-data-between-blocks.md:244]:
```suggestion
1. In the **Evaluate** block, select **value1** and enter `body`. This assigns all the data received from the **Start** block to the `body` variable.
```

File: src/pages/docs/postman-flows/tutorials/building-your-first-flow.md (hunk 4 above)
Comment (jkonrath-postman) [reply src/pages/docs/postman-flows/tutorials/building-your-first-flow.md:65]: Ignore all of these for now, and let's test it or ask Tristan after Monday.

**Review by akinard-postman — COMMENTED** [review]
Just a few minor suggestions!

**Review by jkonrath-postman — APPROVED** [review]
LGTM - add the mini-map one and do any suggested edits of Matthew's and we're good.

File: src/pages/docs/postman-flows/gs/blocks-information.md [hunk 16]
```diff
@@ -41,18 +41,19 @@ _Blocks_ are used to create Flows, and information is passed between them.
...
+* [Working with blocks](#working-with-blocks)
 
 ## Blocks
 
 Blocks are the basic units used to create a flow. Every block has a specific purpose, such as performing actions, creating information, or making a decision.
 
 ### Block fundamentals
 
-Every block has either an input, an output, or both. You can pass a information from one block to another. For more information on each block, see [Types of Blocks](#types-of-blocks).
+Every block has either an input, an output, or both. You can pass information from one block to another. For more information on each block, see [Types of Blocks](#types-of-blocks).
```
[the remark this answers is not in the document]
Comment (petewitcher-postman) [reply src/pages/docs/postman-flows/gs/blocks-information.md]: I think so!

[the remark this answers is not in the document]
Comment (akinard-postman) [reply src/pages/docs/postman-flows/gs/blocks-information.md]: Other than that one last thing LGTM! I'll approve. :)

**Review by akinard-postman — APPROVED** [review]

**Comment by petewitcher-postman** [comment]
Thanks everybody for the excellent comments!

**Pull request merged**